web-app/js/portal/filter/ui/ALASpeciesFilterPanel.js

```diff
@@ -0,0 +1,182 @@
+Ext.namespace('Portal.filter.ui');
+
+Portal.filter.ui.ALASpeciesFilterPanel = Ext.extend(Portal.filter.ui.BaseFilterPanel, {
+
+    constructor: function(cfg) {
+        var config = Ext.apply({
+            autoDestroy: true
+        }, cfg);
+
+        Portal.filter.ui.ALASpeciesFilterPanel.superclass.constructor.call(this, config);
+    },
+
+    _createControls: function() {
+
+        var resultTpl = new Ext.XTemplate(
+            '<tpl for=".">' +
+            '<div class="x-combo-list-item alafilter">',
+            '<div><b>{rawRank}</b> - {highlight} </div>',
+            ' <tpl if="classs != \'\' ">',
+            '  <div><b>Class:</b> {classs}</div>',
+            ' </tpl>',
+            ' <tpl if="phylum != \'\' ">',
+            '  <div><b>Phylum:</b> {phylum}</div>',
+            ' </tpl>',
+            '</div>',
+            '</tpl>'
+        );
+
+        this.jsonStore = new Ext.data.JsonStore({
+            url: 'proxy?', // portal proxy controller
+            root: "searchResults.results",
+            idProperty: 'guid',
+            baseParams : {
+                fq: 'species_habitats:"Marine"', // ALA index for marine only
+                url: "https://biocache-lb.ala.org.au/ws/autocomplete/search"
```

Comment:
As before - this should be in config - especially since we have changed it once already because ALA changed it and need to change it again.